LibNoDaveConnectionLibrary/DataTypes/Projectfolders/Step7V5/BlocksOfflineFolder.cs

```diff
@@ -423,14 +423,14 @@ private TmpBlock GetBlockBytes(string blkName)
            return GetBlockBytes(blkInfo);
         }
 
-        public S7DataRow GetInterface(string blkName)
+        public S7DataRow GetInterface(string blkName, S7ConvertingOptions myConvOpt)

```

Comment:
could you a here a overload without the Converting Options? So it is a non breaking change?

Reply:
yep!
https://github.com/dotnetprojects/DotNetSiemensPLCToolBoxLibrary/blob/169a959f549ed8fb6639134306ba2422b2545431/LibNoDaveConnectionLibrary/DataTypes/Projectfolders/Step7V5/BlocksOfflineFolder.cs#L426-L441